packages/button/package.json

```diff
@@ -1,22 +1,24 @@
 {
   "name": "@pluralsight/ps-design-system-button",
-  "version": "13.0.2",
+  "version": "14.0.0-beta.0",
   "description": "Design System component for button ui",
   "license": "Apache-2.0",
   "repository": "pluralsight/design-system",
   "author": "jaketrent",
-  "main": "index.js",
+  "keywords": [],
+  "main": "react.js",
+  "module": "index.js",
   "scripts": {
-    "build": "run-s build:js build:css",
-    "build:css": "build-css --useGlamor",
-    "build:js": "babel --root-mode upward src --out-dir dist",
+    "build": "run-s build:js build:cjs build:css",
+    "build:cjs": "babel --root-mode upward src --out-dir dist/cjs",
+    "build:css": "build-css --useGlamor -i dist/cjs/css/index.js",
+    "build:js": "cross-env ESM=true babel --root-mode upward src --out-dir dist/esm",
```

Comment:
is the name `build:esm` better? maybe? idk

Reply:
I think the idea is to move every component to esm so I decided to change the name of the old build to reflect it's the fallback `build:cjs`